_posts/international/2020-07-20-2nd-Intl-RSE-Leaders-Call.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Collaborate to start new RSE initiatives at the International RSE Leaders workshop!
+tags: [intertional, event, call]
+posted_by: Daniel S. Katz (for Stephan Druskat)
+---
+
+![Group spelling out RSE](https://research-software.org/img/derse.png)
+Photo by Antonia Cozacu, Jan Philipp Dietrich, de-RSE e.V. (CC BY 4.0).
+
+In 2018, the first International RSE Leaders Workshop, organized by the UK RSE Association, took place at the Alan Turing Institute in London, UK.
+Amongst other successes, it saw the foundation of the Nordic RSE community and helped improve access to software expertise in research.
+
+This year, representatives from existing Research Software Engineering associations - UK RSE, de-RSE, US-RSE, Nordic-RSE, NL-RSE,
+and the newly founded Belgian RSE initiative - are organizing the 2nd International RSE Leaders Workshop.
+The online meeting takes place on 15, 16 and 30 September, and brings together established and aspiring leaders from the
+international Research Software Engineering community for a three-half-days event of networking, knowledge exchange, and hands-on collaboration.
+
+This is the perfect opportunity to meet with key representatives from established RSE communities from around the world.
+The meeting will provide aspiring leaders with the knowledge and contacts needed to establish research software engineering
+in their own countries, and will advance our shared goal of software becoming a first-class research output.
```

Comment:
communities is missing?

Reply:
I agree communities sounds better - but to be consistent with the text posted elsewhere, my feeling is it should probably stay as is. I believe the goal is to post this call all everywhere. @danielskatz?

Reply:
I agree - we can fix this in the others too hopefully